lib/rubocop/runner.rb

```diff
@@ -254,7 +259,7 @@ def enable_rails_cops(config)
     def mobilized_cop_classes(config)
       @mobilized_cop_classes ||= {}
       @mobilized_cop_classes[config.object_id] ||= begin
-        cop_classes = Cop::Cop.all
+        cop_classes = Cop::Cop.all.to_a
```

Comment:
Obviously the problem here is that this returns a `CopStore` as opposed to an array of cops. An API that's not intuitive is not a good API.